bsuite/utils/wrappers_test.py

```diff
@@ -22,13 +22,14 @@
 
 from bsuite.experiments.catch import catch
 from bsuite.utils import wrappers
+from bsuite.utils import gym_wrapper
```

Comment:
Imports should be alphabetical, so this should be after catch and before wrappers.